source/common/common/matchers.cc

```diff
@@ -110,6 +110,14 @@ PathMatcherConstSharedPtr PathMatcher::createPrefix(const std::string& prefix, b
   return std::make_shared<const PathMatcher>(matcher);
 }
 
+PathMatcherConstSharedPtr PathMatcher::createPattern(const std::string& pattern, bool ignore_case) {
```

Comment:
This should be tested in matchers_test.cc.

Reply:
Since it currently behaves like PathMatcher and there's a TODO then maybe it's not too meaningful to add tests at this point. Optional: Maybe make it clear in the TODO that unit tests need to be added for the whole function.

Reply:
It does not perform anything useful yet. Implementation and tests will come in text PR